eclair-core/src/main/scala/fr/acinq/eclair/payment/PaymentInitiator.scala

```diff
@@ -19,18 +19,18 @@ package fr.acinq.eclair.payment
 import java.util.UUID
 import akka.actor.{Actor, ActorLogging, ActorRef, Props}
 import fr.acinq.eclair.NodeParams
-import fr.acinq.eclair.payment.PaymentLifecycle.SendPayment
+import fr.acinq.eclair.payment.PaymentLifecycle.{GenericSendPayment, SendPayment, SendPaymentToRoute}
 
 /**
   * Created by PM on 29/08/2016.
   */
 class PaymentInitiator(nodeParams: NodeParams, router: ActorRef, register: ActorRef) extends Actor with ActorLogging {
 
   override def receive: Receive = {
-    case c: SendPayment =>
+    case c:GenericSendPayment =>
       val paymentId = UUID.randomUUID()
       val payFsm = context.actorOf(PaymentLifecycle.props(nodeParams, paymentId, router, register))
-      payFsm forward c
+      payFsm.forward(c)
```

Comment:
Previous syntaxt was a convention we use elsewhere (to be close to `actor ! msg`)

```suggestion
      payFsm forward c
```